M/Makemsix/build_tarballs.jl

```diff
@@ -0,0 +1,124 @@
+
+    elif [[ "${target}" == "x86_64-w64-mingw32" ]]; then
+        # Windows-specific options
+        find . -name "CMakeLists.txt" -type f -exec sed -i 's/set(CMAKE_CXX_STANDARD 14)/set(CMAKE_CXX_STANDARD 17)/' {} \;
+
+        # Apply Windows-specific patches
+        cp ../src/msix/CMakeLists.txt src/msix/CMakeLists.txt
+        cp ../src/msix/PAL/FileSystem/Win32/DirectoryObject.cpp src/msix/PAL/FileSystem/Win32/DirectoryObject.cpp
+        cp ../src/inc/internal/UnicodeConversion.hpp src/inc/internal/UnicodeConversion.hpp
+        cp ../src/inc/public/MSIXWindows.hpp src/inc/public/MSIXWindows.hpp # optional, eliminates warnings
+
+        sed -i 's/static constexpr const IID IID_##name/inline constexpr const IID IID_##name/' src/inc/public/AppxPackaging.hpp
+        sed -i 's/#ifdef WIN32/#if defined(WIN32) \&\& !defined(__MINGW32__)/' src/inc/public/AppxPackaging.hpp
+        sed -i 's/const PfnDliHook __pfnDliFailureHook2 = MsixDelayLoadFailureHandler;//' src/msix/common/Exceptions.cpp
+
+        # Fix case-sensitive includes for Windows
+        find src -type f \( -name "*.c" -o -name "*.cpp" -o -name "*.h" -o -name "*.hpp" \) -exec sed -i 's/#include "UnKnwn.h"/#include "unknwn.h"/g' {} \;
+        find src -type f \( -name "*.c" -o -name "*.cpp" -o -name "*.h" -o -name "*.hpp" \) -exec sed -i 's/#include "Unknwn.h"/#include "unknwn.h"/g' {} \;
+        find src -type f \( -name "*.c" -o -name "*.cpp" -o -name "*.h" -o -name "*.hpp" \) -exec sed -i 's/#include "Objidl.h"/#include "objidl.h"/g' {} \;
+
+        # Windows-specific compiler flags
+        export LDFLAGS="-L${libdir}"
+        export CXXFLAGS="$CXXFLAGS -include $WORKSPACE/srcdir/src/inc/compat/msix_win_api_fixes.h"
+
+        PLATFORM_OPTIONS="-DWIN32=on"
+    fi
+
+    # Run cmake with common options + platform-specific options
+    cmake -B build \
+        -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+        -DCMAKE_BUILD_TYPE=MinSizeRel \
+        -DSKIP_BUNDLES=on \
+        -DUSE_VALIDATION_PARSER=on \
+        -DMSIX_PACK=on \
+        -DMSIX_SAMPLES=on \
+        -DMSIX_TESTS=off \
+        -DXML_PARSER=xerces \
+        -DCRYPTO_LIB=openssl \
+        -DCMAKE_PREFIX_PATH=${prefix} \
+        -DBUILD_SHARED_LIBS=ON \
+        -DUSE_MSIX_SDK_ZLIB=on \
+        -DOPENSSL_ROOT_DIR="${prefix}" \
+        -DOPENSSL_CRYPTO_LIBRARY=${libdir}/libcrypto.${dlext} \
+        -DOPENSSL_SSL_LIBRARY=${libdir}/libssl.${dlext} \
+        ${PLATFORM_OPTIONS} \
+        .
+    
+    # Build only what we need
+    make -j${nproc} -C build/src
+
+    # Install license and binary
+    install_license LICENSE 
+    install -Dvm 755 "build/bin/makemsix${exeext}" "${bindir}/makemsix${exeext}"
+
+    # Platform-specific library installation
+    if [[ "${target}" == "x86_64-w64-mingw32" ]]; then
+        install -Dvm 755 build/lib/libmsix.* "${libdir}/"
+        install -Dvm 755 build/bin/libmsix.* "${libdir}/"
+        install -Dvm 755 build/bin/libmsix.* "${bindir}/"
```

Comment:
One of these lines is redundant (the first one maybe?)

Reply:
I kept the first one.